database/migrations.go

```diff
@@ -496,4 +496,12 @@ var migrations = []func(tx *sql.Tx) error{
 		`)
 		return err
 	},
+		func(tx *sql.Tx) (err error) {
```

Comment:
If you don't mind keeping the func line aligned with others. It's cosmetic but I like consistence.

Reply:
done, I thought `go fmt` would take care of that and didn't look further :/